### ISSUE TITLE
Contribute to this repo!!

### ISSUE THREAD

**Issue by Agnik7** [Agnik7/Arduino-Codes#9]
This is an open source repository made by the community, for the community!!!!

Feel like contributing? Take a minute to look at the [Contribution Guidelines](https://github.com/Agnik7/Arduino-Codes/blob/main/Contributing.md) and just simply follow the steps mentioned.

Kindly take a moment to star this repository as well, so that it reaches to more and more people.

While raising a PR, just tag this issue, and if merged, the PR will be accepted under Hacktoberfest.

Happy Coding!!!!

**Comment by Tejoooo** [comment]
can u assign me to this issue

**Comment by Agnik7** [comment]
Hi @Tejoooo , I am sorry for the inconvenience, however, I cannot assign this issue to you. This is an open issue for anyone to contribute. You can still contribute, by just mentioning this issue in your PR. Have a nice day!!!

**Comment by Tejoooo** [comment]
Yeah Ohk.Will I be getting the hacktoberfest-accepted if u do merge the pr after the contributions

**Comment by Agnik7** [comment]
Yes
